cumulus_library/databases/athena.py

```diff
@@ -32,19 +32,22 @@ def __init__(self, region: str, work_group: str, profile: str, schema_name: str)
         self.work_group = work_group
         self.profile = profile
         self.schema_name = schema_name
+        self.connection = None
+
+    def connect(self):
         # the profile may not be required, provided the above three AWS env vars
         # are set. If both are present, the env vars take precedence
```

Comment:
This comment is lightly out of date now, since it refers to "above three AWS env vars" and it got moved around